### PR TITLE
kubeadm: include the 'certs renew' sub-command in the CLI

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:

The new 'renew' command seems to be missing in the CLI.
Add it as sub-command under 'kubeadm alpha phase certs'.

to follow up on these series:
https://github.com/kubernetes/kubernetes/commits/master/cmd/kubeadm/app/cmd/phases/certs/renew.go

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

/priority critical-urgent
/kind bug
@kubernetes/sig-cluster-lifecycle-pr-reviews 
/assign @liztio @timothysc

### CODE REVIEW

File: cmd/kubeadm/app/cmd/phases/certs.go [hunk 1]
```diff
@@ -120,11 +121,14 @@ func getCertsSubCommands(defaultKubernetesVersion string) []*cobra.Command {
...
 
-	subCmds := []*cobra.Command{allCmd, saCmd}
+	// "renew" command
+	renewCmd := certscmdphase.NewCmdCertsRenewal()
 
+	// add the rest of commands
 	certTree, err := certsphase.GetDefaultCertList().AsMap().CertTree()
 	kubeadmutil.CheckErr(err)
 
+	subCmds := []*cobra.Command{allCmd, saCmd, renewCmd}
```
Comment (xiangpengzhao) [inline cmd/kubeadm/app/cmd/phases/certs.go]: Personally it's more clear to move this line after L125. But it's ok as-is.

Comment (neolit123) [reply cmd/kubeadm/app/cmd/phases/certs.go]: i will move this:
```
certTree, err := certsphase.GetDefaultCertList().AsMap().CertTree()
kubeadmutil.CheckErr(err)
```
towards the beginning of the function instead,.

if it fails there is no point to attach the commands, even.

Comment (neolit123) [reply cmd/kubeadm/app/cmd/phases/certs.go]: actually this isn't good either.
will revert the change to reduce the noise.

**Review by timothysc — APPROVED** [review]

**Comment by neolit123** [comment]
/assign @smarterclayton 

for `docs/` approve.

**Pull request merged**